operator/src/main/java/dev/responsive/k8s/operator/reconciler/ResponsivePolicyReconciler.java

```diff
@@ -74,27 +80,54 @@ PolicyType.KAFKA_STREAMS, new KafkaStreamsPolicyPlugin(environment)
     this.plugins = Objects.requireNonNull(plugins);
   }
 
+  private Optional<ApplicationState> pollTargetState(final ResponsivePolicy policy) {
+    try {
+      return Optional.of(responsiveCtx.getControllerClient().getTargetState(
+          ControllerProtoFactories.emptyRequest(environment, policy)));
+    } catch (final StatusRuntimeException e) {
+      if (e.getStatus().getCode().equals(Status.NOT_FOUND.getCode())) {
+        LOG.info("no target state found");
```

Comment:
these are going to be really spammy right? something we should fix with the operator is that today when there's no target state we have spammy exceptions logs which make it feel like something is wrong. I think it's probably OK not to log this here and instead log when it does find something.

Reply:
yeah that makes sense. I'll drop this